internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -178,6 +179,19 @@ func (r *BMACReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		}
 	}
 
+	// For SNO, Ironic should not manage the node after it has been installed
+	// or if installation is in progress. At least for 4.8, it should be left
+	// to the hub to manage it. To do so, we need to set the BMH to detached state
```

Comment:
what doesn't mean "should be left to the hub to manage it" ?

Reply:
and why it differs from NON-SNO ? If we have started the installation we should set detached right? cause reboot will occur from the installer side

Reply:
@rwsu for sno, it means that it will not be managed anymore. and for rwn, it means that the spoke cluster will manage it. 
The comment is incorrect.

Reply:
I've remove any mention of SNO. The logic should look only at agent status now. Comment updated.